lib/glueby.rb

```diff
@@ -1,6 +1,12 @@
 require "glueby/version"
 require 'tapyrus'
 
+begin
+  require 'rails'
```

Comment:
Why require `rails`? It doesn't seem to be defined in the gemspec.

Reply:
I was thinking that it is necessary if glueby is loaded in a rails project. Glueby needs to use Rails::Railtie module. However, I didn't confirm whether it causes some error or not. I will check it.

Reply:
I removed these codes. There is no error if it is removed.